### ISSUE TITLE
feat: add methods to get the current reqId, and get data saved on proxyReqs for that reqId

### ISSUE THREAD

**Issue by izyak** [icon-project/IBC-Integration#747]
Currently, the xcall contract doesn't have read method for what the next `reqId` is, and if executeCall has been called for any arbitrary `reqId`. Though we can explore the tracker for this info, it'll be quite cumbersome, and it might make more sense to add methods in contract for these 2 things.

**Comment by pragyanshrestha-ibriz** [comment]
Might be resolved by xcall tracker. Icebox for now.

**Comment by sharma66mahesh** [comment]
Yes I think this feature would be necessary for the frontend to track status of the xcall (assuming the realtime event listener will not always be listening for respective events on the destination chain)

**Comment by izyak** [comment]
From the support side, we can only check this in the xcall-tracker. But, for application developers, it seems this will be necessary unless the api for xcall-tracker will be open. Nonetheless, we should add this in the next update of xcall.
